perception/elevation_map_loader/include/elevation_map_loader/elevation_map_loader_node.hpp

```diff
@@ -97,7 +98,7 @@ class ElevationMapLoaderNode : public rclcpp::Node
   bool use_inpaint_;
   float inpaint_radius_;
   bool use_elevation_map_cloud_publisher_;
-  pcl::shared_ptr<grid_map::GridMapPclLoader> grid_map_pcl_loader_;
+  std::string param_file_path;
```

Comment:
```suggestion
  std::string param_file_path_;
```

Reply:
or not making a member, using  `this->declare_parameter("param_file_path", "path_default");` in `createElevationMap()` may be good